tests/page-objects/messages/sms-gateway.po.js

```diff
@@ -0,0 +1,114 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.po');
+const helper = require('../../helper');
+const { browser, element } = require('protractor');
+
+//page objects        
+const  sentTask= element(by.css('#reports-content .details > ul .task-list .task-state .state'));
+   
+const deliveredTask = element(by.css(
```

Comment:
As a NIT, Consider a function that takes in indexes and builds the css selector for more reuse.